### ISSUE TITLE
Has this work been published？

### ISSUE THREAD

**Issue by YuhuaXu** [JohnTian/DepthImageUpsampling#2]
Has this work been published？

**Comment by Digital2Slave** [comment]
Yes. In Chinese version [Depth upsampling algorithm based on image edge characteristics](http://www.joelcn.com/ch/reader/view_abstract.aspx?file_no=20141219&flag=1)

**Comment by sandeep661** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://link.bnbreward.online/0mhPJ2)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://link.bnbreward.online/0mhPJ2) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @ArdaOzcan, @rinr4t, @lucastrvsn, @JDDevHub, @xieguanliang-yantu, @ivanmlerner, @elliotpryde, @Benbinbin, @the-code-innovator, @osgliath, @HLT, @federerfanatic, @SunHawk999, @Manel-Bel, @MarcBollmann, @Posandu, @pranavkarthik10, @IsaacMAllen, @baizhi1989, @mrjohannchang, @nakulz, @lxwulf, @KingsleyYoo, @brunobalic, @qducnguyen, @Vermill1on, @M1n35Rtvg, @iamsahebgiri, @rickywrx, @Lukasz-Pe, @deekappyok, @3eesa54, @afx1, @diego1araujo, @GabeDuarteM, @w3bh0ok, @Zhang-Jiayong, @DanSpicyTaco, @vuhoanghuynh, @mechabrhma, @DivX0, @multlurk, @nikklavzar, @kizmo, @kuznya, @dbishoponline, @dec0dOS, @r3b00tl00p, @kohey18, @emre-sky
